platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/components/ShadowgraphWiring.java

```diff
@@ -58,6 +58,6 @@ public static ShadowgraphWiring create(@NonNull final TaskScheduler<Void> taskSc
     public void bind(@NonNull final Shadowgraph shadowgraph) {
         ((BindableInputWire<EventImpl, Void>) eventInput).bind(shadowgraph::addEvent);
         ((BindableInputWire<NonAncientEventWindow, Void>) nonExpiredEventWindowInput)
```

Comment:
nit: the bound method changed to `updateEventWindow`, perhaps the member `nonExpiredEventWindowInput` should receive the same update

Reply:
changed name to `eventWindowInput`